electron/launcher/src/components/Registration/Registration.tsx

```diff
@@ -1,7 +1,9 @@
 import React, {useEffect} from 'react';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
 import { userActions } from '_actions';
 import FormBuilder from 'components/FormBuilder';
+import logo from '../../assets/dietstory_logo.png';
```

Comment:
Again absolute path from src, like this: `import logo from 'assets/dietstory_logo.png';`